sim/midas/src/main/scala/midas/models/dram/FASEDMemoryTimingModel.scala

```diff
@@ -22,6 +22,10 @@ import Console.{UNDERLINED, RESET}
 
 import java.io.{File, FileWriter}
 
+object ModelID {
```

Comment:
I mentioned this earlier but we should reuse the ID in Widget.scala. We could add a method, `getInstanceID` there.